lms/djangoapps/certificates/models.py

```diff
@@ -842,9 +842,9 @@ def course_key(self):
 
 
 class CertificateGenerationCourseSetting(TimeStampedModel):
-    """Enable or disable certificate generation for a particular course.
+    """Enable or disable certificate settings for a particular course.
 
-    This controls whether students are allowed to "self-generate"
+    'enabled' controls whether students are allowed to "self-generate"
```

Comment:
Could you rename this field to something more descriptive, like `can_self_generate` or something? You should be able to do that via a migration. Just make sure you also update all the places in the code that look at that value.

Reply:
Definitely, does 'learner_certificate_generation' seem like a reasonable enough name? Also, I think I can only do this by adding a new field with the new name, and copying over the data from the old column,  as per https://openedx.atlassian.net/wiki/spaces/OpenDev/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Makingamigrationtomodifyanexistingtable